packages/navie/src/agents/generate-agent.ts

```diff
@@ -55,7 +152,12 @@ export default class GenerateAgent implements Agent {
     const agentPrompt = [GENERATE_AGENT_PROMPT];
     // With the /noformat option, the user will explain the desired output format in their message.
     if (options.userOptions.isEnabled('format', true)) {
-      agentPrompt.push(GENERATE_AGENT_FORMAT);
+      agentPrompt.push(
+        // see if the new format is requested
+        options.userOptions.isEnabled('format-change', false)
```

Comment:
I suggest `/format=xml`, this would align with some other agents that accept a format argument.

Reply:
Done!